### PR TITLE
Add a name to the program

### PR DESCRIPTION
**Pull request by ztao-8**
#1 fix first issue

### CODE REVIEW

File: HelloWorld.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 public class HelloWorld {
     public static void main(String[] args) {
         System.out.println("Hello, World!");
+        System.out.println("Kexin!");
```
Comment (SkyeKD) [inline HelloWorld.java:4]: add comment

**Review by SkyeKD — CHANGES_REQUESTED** [review]
add comment

File: HelloWorld.java (hunk 1 above)
Comment (ztao-8) [reply HelloWorld.java:4]: I added it

**Review by SkyeKD — APPROVED** [review]
looks good

**Pull request merged**